portfolio/src/main/webapp/index.html

```diff
@@ -1,276 +1,316 @@
+                                class="ri-linkedin-fill"></i></a></li>
+                </ul>
+                <!-- social -->
             </div>
         </div>
     </header>
     <!-- header -->
     <!-- banner -->
     <div class="ds-banner">
-      <div class="container" style="margin-top: 180px;">
-        <div class="row">
-          <div class="col-12 col-sm-12 col-md-5 col-lg-5 col-xl-5 col-xxl-5">
-            <figure>
-              <img src="/images/profile.jpg">
-            </figure>
-          </div>
-          <div class="col-12 col-sm-12 col-md-7 col-lg-7 col-xl-7 col-xxl-7">
-            <section>
-                <h1>
-                  <span>HI THERE</span>
-                  I’m Ananya Aggarwal <br> SOFTWARE ENGINEER + STUDENT located in Wisconsin.
-                </h1>
-            </section>
-          </div>
+        <div class="container" style="margin-top: 180px;">
+            <div class="row">
+                <div class="col-12 col-sm-12 col-md-5 col-lg-5 col-xl-5 col-xxl-5">
+                    <figure>
+                        <img src="/images/profile.jpg">
+                    </figure>
+                </div>
+                <div class="col-12 col-sm-12 col-md-7 col-lg-7 col-xl-7 col-xxl-7">
+                    <section>
+                        <h1>
+                            <span>HI THERE</span>
+                            I’m Ananya Aggarwal <br> SOFTWARE ENGINEER + STUDENT located in Wisconsin.
+                        </h1>
+                    </section>
+                </div>
+            </div>
         </div>
-      </div>
     </div>
     <!-- banner -->
 
     <!-- about -->
     <div class="ds-about-section" id="about-section">
-      <div class="container">
-        <section>
-            <h2 class="ds-heading">About Me</h2>
-            <p id="about">I am a rising sophmore at Univeristy of Wisconsin, Madison majoring in Computer Science. I am an aspiring software engineer. I love spending time fixing little details and optimizing Programming applications. I am passionate about learning and building new things every day which is why I love Computer Science. </p>
-            <div class="ds-button-sec text-center">
-            <a href="#" class="ds-button" >Download Resume</a>
-            </div>
-        </section>
-      </div>
+        <div class="container">
+            <section>
+                <h2 class="ds-heading">About Me</h2>
+                <p id="about">I am a rising sophmore at Univeristy of Wisconsin, Madison majoring in Computer Science. I
+                    am an aspiring software engineer. I love spending time fixing little details and optimizing
+                    Programming applications. I am passionate about learning and building new things every day which is
+                    why I love Computer Science. </p>
+                <div class="ds-button-sec text-center">
+                    <a href="#" class="ds-button">Download Resume</a>
```

Comment:
This link won't work as of now, I assume you're planning to link something there in the future? Maybe put a 
`<!-- TODO: Fix link to resume -->` in there